django_slack_oauth/pipelines.py

```diff
@@ -0,0 +1,14 @@
+from django_slack_oauth.models import SlackOAuthRequest
+
+
+def log_request(request, api_data):
+        SlackOAuthRequest.objects.create(
+            access_token=api_data.pop('access_token'),
+            extras=api_data
```

Comment:
Remove ? :)
Since you added IP field in the model you could just add this on creation.